### PR TITLE
feat: Improve clarity of Integrations page

### LINKED ISSUE #5875: Improve clarity of Integrations page
This issue is part of the efforts captured #5843.

There is some confusion around integrations, [particularly about the personal vs. team scope](https://github.com/ParabolInc/parabol/issues/5533).

We also have an opportunity to improve the overall clarity of the integrations page, with what I consider to be small, low-effort updates.

## What I'm proposing

image

**Better messaging and descriptions**
We can add a short descriptive line at the top of the integrations page that clarifies the scope of the integrations in Parabol. I' proposing `Integrations are personal and specific to the team where you add them.` but I'm looking for suggestions from you. 

The goal of this text is bring clarify to the confusion reported in #5533.

Along with this, we can clarify what integrations are active vs. which ones are available. An easy way to do this is to separate them by category:

Captura de Pantalla 2022-01-17 a la(s) 12 15 03 p m

Lastly, we can also improve the clarity of the status of the active integrations by adding a `Connected` label instead of just showing an icon like we currently do. We can also use this space to display errors when needed.

Captura de Pantalla 2022-01-17 a la(s) 12 18 14 p m

**Bonus points:**

And since we're already tweaking the layout here, let's align the ellipsis button and make it look like a button so everything is more polished. 💅

### Estimated effort:
- 16 hrs

### LINKED ISSUE #6797: Change input placeholder text color
Placeholder text color feels too strong, which might be confusing:

I suggest to pick another color:

image

AC:

- Change input placeholder color so that it cannot be confused with plain text

**Estimated effort:** 1 points

### PR DESCRIPTION
**Pull request by igorlesnenko**
Fixes #5875
Fixes #6797

image

**How to test:**

- Make all integrations visible (Add feature flags for Azure DevOps and MS Teams, add Azure DevOps provider)
- Check that all integrations can be connected and all controls are working as expected
- Check mobile markup

### CODE REVIEW

**Review by enriquesanchez — APPROVED** [review]
I can't speak for everything else, but the new placeholder text for the webhooks looks good to me 👍

**Comment by enriquesanchez** [comment]
Two additional notes, but **these can be handled on separate PRs**:

1. Can we increase the spacing around the description text and the Connected and Available sub-headings? You can reference the mockup on https://github.com/ParabolInc/parabol/issues/5875.

2. Where you able to [look at this comment](https://github.com/ParabolInc/parabol/issues/5875#issuecomment-1133093531) from that same issue?

>Add a See how it works link to the description of each integration that links to a support/help document that highlights how to use it.

**Comment by igorlesnenko** [comment]
> Where you able to https://github.com/ParabolInc/parabol/issues/5875#issuecomment-1133093531 from that same issue?
@enriquesanchez yeah, I added the link. Does it look correct?

image

File: packages/client/modules/teamDashboard/components/ProviderRow/ProviderRow.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+              ) : (
+                <MenuButton onClick={togglePortal} ref={menuRef}>
+                  <Icon>more_vert</Icon>
+                </MenuButton>
+              )}
+            </>
+          )}
+        </ProviderActions>
+      </CardTop>
+      {children}
```
Comment (nickoferrall) [inline packages/client/modules/teamDashboard/components/ProviderRow/ProviderRow.tsx:192]: +2 great work making this reusable so we can save a ton of CSS in the other components!

File: packages/client/modules/teamDashboard/components/ProviderRow/ProviderRow.tsx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
+  return (
+    <ProviderCard>
+      <CardTop>
+        {providerLogo}
+        <RowInfo>
+          <ProviderName>{providerName}</ProviderName>
+          <RowInfoCopy>
+            {providerDescription}.{' '}
+            <a href={seeHowItWorksUrl} target='_blank' rel='noreferrer'>
```
Comment (nickoferrall) [inline packages/client/modules/teamDashboard/components/ProviderRow/ProviderRow.tsx:142]: +1 I'd make this text a bit bolder. I actually didn't notice it was a hyperlink until I looked on mobile

**Review by nickoferrall — APPROVED** [review]
This looks great - I really like separating the Connected vs Available providers!

**Comment by igorlesnenko** [comment]
@enriquesanchez @nickoferrall Updated the layout:

image

**Comment by igorlesnenko** [comment]
See how it works URLs will be updated with correct support pages

**Review by Dschoordsch — CHANGES_REQUESTED** [review]

File: packages/client/modules/teamDashboard/components/ProviderRow/StyledButton.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import styled from '@emotion/styled'
+import FlatButton from '../../../../components/FlatButton'
+import {PALETTE} from '../../../../styles/paletteV3'
+
+const StyledButton = styled(FlatButton)({
```
Comment (Dschoordsch) [inline packages/client/modules/teamDashboard/components/ProviderRow/StyledButton.tsx:5]: +1 Could get a better name, like in what way is it styled differently than `FlatButton`?

**Review by Dschoordsch — APPROVED** [review]
Did not test again, but code looks good!

**Pull request merged**